src/apps/content-editor/src/app/views/LinkCreate/LinkCreate.js

```diff
@@ -17,7 +17,7 @@ import CardActions from "@mui/material/CardActions";
 
 import { FieldTypeInternalLink } from "@zesty-io/core/FieldTypeInternalLink";
 import { FieldTypeText } from "@zesty-io/material";
-import { FieldTypeUrl } from "@zesty-io/core/FieldTypeUrl";
+import { FieldTypeUrl } from "@zesty-io/material";
```

Comment:
Watch out for duplicate imports

This should be 

import { FieldTypeText, FieldTypeUrl } from "@zesty-io/material";